deploy.sh

```diff
@@ -15,30 +15,23 @@ else
     exit 1
 fi
 
-# tell the user what's going on
-echo "Building & deploying project..."
-
-# get ID of current commit
-commitHash=$(git rev-parse HEAD) &&
-commitAuthor=$(git --no-pager show -s --format='%an <%ae>' HEAD) &&
-# add a comment to the html that indicates the current commit
-sed -i "4i\<!--This site was built against the source of the following commit: $commitHash ($commitAuthor) -->" ./src/main/webapp/index.html &&
-sed -i "4i\<!--This site was built against the source of the following commit: $commitHash ($commitAuthor) -->" ./src/main/webapp/sequenceBuilder.html &&
-gitTagInserted=true
-
-# build project
+# run frontend build tools
+echo "Building frontend..." &&
+if [ "${2}" == "x" ]
+then
+    # build for production (no react dev tools, optimized performance)
+    npm run build-prod
+else
+    # build for development (with react dev tools, bad performance)
+    npm run build-dev
+fi &&
+
+# compile backend sources and package frontend assets
+echo "Building backend..." &&
 mvn clean install -q &&
-mvnCleanInstallSuccessful=true
-
-if $gitTagInserted
-then # remove git tag
-    # we do not want our deploy script to change our source files,
-    # only the deployed version of them
-    sed -i "4d" ./src/main/webapp/index.html &&
-    sed -i "4d" ./src/main/webapp/sequenceBuilder.html
-fi
+projectBuildSuccessful=true
```

Comment:
I think the bug existed earlier but I appreciate you taking the blame :+1: .  Not to be a bother but could you plz give me a little extra detail about how to reproduce the problem so that I can fix it.  Thx bb :cupid: